packages/developer-portal/src/components/ui/developer-invite-member-modal/developer-invite-member-modal.tsx

```diff
@@ -50,6 +51,14 @@ export const InviteMemberModalInput: React.FC = () => {
         required
         labelText={inviteEmailField.label as string}
       />
+      <Input
+        id={inviteJobTitleField.name}
+        type="text"
+        placeholder={inviteJobTitleField.placeHolder}
+        name={inviteJobTitleField.name}
+        required
+        labelText={inviteJobTitleField.label as string}
```

Comment:
Why need to make it `as string`? form field label should be typed string already

Reply:
Others LGTM